configs/tasks/rearrange/rearrange_easy_multi_agent.yaml

```diff
@@ -0,0 +1,169 @@
+          AGENT: 0
+          TYPE: BaseVelAction
+          LIN_SPEED: 10.0
+          ANG_SPEED: 10.0
+          ALLOW_DYN_SLIDE: true
+          END_ON_STOP: false
+          ALLOW_BACK: true
+          MIN_ABS_LIN_SPEED: 1.0
+          MIN_ABS_ANG_SPEED: 1.0
+      AGENT_0_PDDL_APPLY_ACTION:
+          AGENT: 0
+          TYPE: PddlApplyAction
+      AGENT_1_ARM_ACTION:
+          AGENT: 1
+          TYPE: ArmAction
+          ARM_CONTROLLER: ArmRelPosAction
+          GRIP_CONTROLLER: MagicGraspAction
+          ARM_JOINT_DIMENSIONALITY: 7
+          GRASP_THRESH_DIST: 0.15
+          DISABLE_GRIP: false
+          DELTA_POS_LIMIT: 0.0125
+          EE_CTRL_LIM: 0.015
+      AGENT_1_BASE_VELOCITY:
+          AGENT: 1
+          TYPE: BaseVelAction
+          LIN_SPEED: 10.0
+          ANG_SPEED: 10.0
+          ALLOW_DYN_SLIDE: true
+          END_ON_STOP: false
+          ALLOW_BACK: true
+          MIN_ABS_LIN_SPEED: 1.0
+          MIN_ABS_ANG_SPEED: 1.0
+      AGENT_1_PDDL_APPLY_ACTION:
+          AGENT: 1
+          TYPE: PddlApplyAction
+      AGENT_0_REARRANGE_STOP:
+          TYPE: RearrangeStopAction
+      AGENT_1_REARRANGE_STOP:
+          TYPE: RearrangeStopAction
+    POSSIBLE_ACTIONS:
+      - AGENT_0_ARM_ACTION
+      - AGENT_0_BASE_VELOCITY
+      - AGENT_0_PDDL_APPLY_ACTION
+      - AGENT_0_REARRANGE_STOP
+      - AGENT_1_ARM_ACTION
+      - AGENT_1_BASE_VELOCITY
+      - AGENT_1_PDDL_APPLY_ACTION
+      - AGENT_1_REARRANGE_STOP
+SIMULATOR:
+    ADDITIONAL_OBJECT_PATHS:
+        - "data/objects/ycb/configs/"
+    ACTION_SPACE_CONFIG: v0
+    GRASP_IMPULSE: 10000.0
+    HOLD_THRESH: 0.2
+    AC_FREQ_RATIO: 4
+    DEBUG_RENDER: false
+    DEBUG_RENDER_GOAL: true
+    AGENTS:
+    - AGENT_0
+    - AGENT_1
+    AGENT_0:
+        HEIGHT: 1.5
+        RADIUS: 0.3
+        IS_SET_START_STATE: false
+        SENSORS:
+        - HEAD_DEPTH_SENSOR
+        ROBOT_URDF: ./data/robots/hab_fetch/robots/hab_suction.urdf
+        ROBOT_TYPE: FetchSuctionRobot
+        IK_ARM_URDF: ./data/robots/hab_fetch/robots/fetch_onlyarm.urdf
+        JOINT_START_NOISE: 0.0
+    AGENT_1:
+        HEIGHT: 1.5
+        RADIUS: 0.3
+        IS_SET_START_STATE: false
+        SENSORS:
+        - HEAD_DEPTH_SENSOR
+        ROBOT_URDF: ./data/robots/hab_fetch/robots/hab_suction.urdf
+        ROBOT_TYPE: FetchSuctionRobot
+        IK_ARM_URDF: ./data/robots/hab_fetch/robots/fetch_onlyarm.urdf
+        JOINT_START_NOISE: 0.0
+    HEAD_RGB_SENSOR:
+        WIDTH: 256
+        HEIGHT: 256
+    HEAD_DEPTH_SENSOR:
+        WIDTH: 256
+        HEIGHT: 256
+        MIN_DEPTH: 0.0
+        MAX_DEPTH: 10.0
+        NORMALIZE_DEPTH: true
+    CTRL_FREQ: 120.0
+    DEFAULT_AGENT_ID: 0
+    FORWARD_STEP_SIZE: 0.25
+    HABITAT_SIM_V0:
+        ALLOW_SLIDING: false
+        ENABLE_PHYSICS: true
+        GPU_DEVICE_ID: 0
+        GPU_GPU: false
+        PHYSICS_CONFIG_FILE: ./data/default.physics_config.json
+    SEED: 100
+    TYPE: RearrangeSim-v0
```

Comment:
Can we add some test configs for baselines to [test config dir](https://github.com/facebookresearch/habitat-lab/tree/main/habitat_baselines/config/test) so it get test automatically.

Reply:
Hmm I don't think we should include any Baselines trainer setup in this PR. My trainer code for the multi-agent setup is very different.